FrontEnd/main/src/App.js

```diff
@@ -5,8 +5,11 @@ import MainPartners from "./components/landing-page/partners/Partners";
 import MainCompanies from "./components/landing-page/companies/Companies";
 import MainLoginBanner from "./components/landing-page/login-banner/LoginBanner";
 import MainAboutSection from "./components/landing-page/about-section/About";
+import CookieMod from './components/cookieacception/CookieMod';
+import { useState } from 'react';
```

Comment:
nit: We usually keep react import at the top of document

Reply:
I'll remember it. Thanl you. Done